scala/scala.bzl

```diff
@@ -786,6 +803,8 @@ def scala_repositories():
 
   native.bind(name = "io_bazel_rules_scala/dependency/scalatest/scalatest", actual = "@scalatest//jar")
 
+# With providers changes, this is completely deprecatable once bazel release includes
+# fixes to java rules using JavaProvider
```

Comment:
It's pretty hard to see what is going on with bazel development without also figuring out the gerrit thing. I get that you all use it internally, but I never have and honestly it feels like some monstrous UI from 1993 or something. I am dreaming of the day all PRs go to github.